tools/build_protobuf

```diff
@@ -71,6 +72,7 @@ do_rebuild() {
 do_check() {
     # rebuild protobuf in tmpdir and check result against specified directory
     local TMPDIR=$(mktemp -d proto-check.XXXXXX)
+    trap "rm -r $TMPDIR" RETURN
```

Comment:
```
In tools/build_protobuf line 75:
    trap "rm -r $TMPDIR" RETURN
                ^-----^ SC2064: Use single quotes, otherwise this expands now rather than when signalled.
```

Reply:
AFAICT the usage is correct -- TMPDIR never changes so I am perfectly OK with expanding now
or do you see another issue?

Reply:
We want to have it expanded now, since the variable is local

Reply:
(it's not a problem just fyi)

Reply:
but it traps `RETURN` so on function exit so this should exist anyway
interesting point though. would `trap RETURN` run in the function scope?

Reply:
We can add `# shellcheck disable=SC2064` on the preceding line